howto/android/custom-vhal.md

```diff
@@ -136,6 +136,100 @@ addons:
   - custom-vhal
 ```
 
+# Integrate Anbox VHAL interface
+
+To use and integrate the [Anbox VHAL interface](https://github.com/canonical/vendor_canonical_interfaces/tree/main/vehicle) into a VHAL implementation, the implementation must be built against Android 14. Before proceeding, ensure you have initialized the Android 14 manifest repository. If not, please follow the [official documentation](https://source.android.com/docs/setup/download) to set it up.
+
+1. Add a new remote definition named `github` to `.repo/manifests/manifest.xml` file.
+
+       <remote name="aosp"
+          fetch=".."
+          review="https://android-review.googlesource.com/" />
+
+       <remote name="github"
+          fetch="https://github.com/canonical/" />
+
+1. Add a new project named `vendor_canonical_interfaces` to the newly added remote and set it to use the `main` branch.
+
+       <project path="vendor/canonical/interfaces"
+                name="vendor_canonical_interfaces"
+                remote="github"
+                revision="main" />
+
+1. Sync the project with the remote
+
+      $ repo sync vendor_canonical_interfaces
+
+1. In the VHAL implementation, create a VHAL manifest fragment named `vendor.canonical.interfaces.vehicle@1.0.xml`.
+
+       <manifest version="1.0" type="device">
+           <hal format="hidl">
+               <name>vendor.canonical.interfaces.vehicle</name>
+               <transport>hwbinder</transport>
+               <fqname>@1.0::IVehicle/default</fqname>
+           </hal>
+       </manifest>
+
+   Place the VHAL manifest fragment file in the same folder as the `Android.bp` file that declares the VHAL service, and include the Anbox VHAL manifest fragment in the VHAL service declaration within the `Android.bp` file. Additionally, add the HIDL module as a shared library that the VHAL service links to in the same file.
```

Comment:
```suggestion
   Place the VHAL manifest fragment file in the same folder as the `Android.bp` file that declares the VHAL service, and include the Anbox VHAL manifest fragment in the VHAL service declaration within the `Android.bp` file. Additionally, add the HIDL module as a shared library that the VHAL service links to, in the same VHAL manifest fragment file.
```

Reply:
hmm, no.
This refers to the Android.bp file, not the device manifest fragment file.
By simply removing "in the same file" the sentence still maintains its meaning but flows more smoothly.
How does that sound?

Reply:
Thanks, that's the kind of clarity we need in such technical context. I think it doesn't hurt to mention the `android.bp` file.